content/docs/contributor/peterportal/getting-started.mdx

```diff
@@ -59,6 +59,8 @@ This document will guide you through setting up the project on your local machin
 
 1. Inside the `api` directory, make a copy of the `.env.example` file and name it `.env`
 2. Add additional variables as needed for features requiring authentication or database access.
+   - You may want an Anteater API key; see [the Anteater API docs](https://docs.icssc.club/docs/developer/anteaterapi/keys-limits) for more information.
```

Comment:
you may want to specify in text here that it's for rate limit purposes, though you could also omit this since it may cause more confusion and doesn't really make that large of a difference unless you need search, which is the next bullet point.